eng/ci/templates/official/jobs/run-non-e2e-tests.yml

```diff
@@ -3,12 +3,12 @@ jobs:
   displayName: Run Non-E2E Integration Tests
 
   pool:
-    name: 1es-pool-azfunc
-    image: 1es-windows-2022 
+    name: 1es-pool-azfunc-large
```

Comment:
What is the significance of this "-large" suffix for the name?

Reply:
I was seeing is test flakiness was due to tight timing issues. So, I created another VM pool which has 8 core machines instead of 2. I will revert and re-test this without the large VM pool.